### PR TITLE
Pol 167 update saa s policies for credential service

### PR DESCRIPTION
**Pull request by KTebrewal**
### Description

Update listed SaaS policy to support the credential service. The changes include-
-Replace auth declaration with the credential declaration.
-Remove long_description field and use new info hash for Version, providers and service.
-Update readme to replace credential instructions with instructions to use credential service.
-Update CWF definitions as needed.

### Issues Resolved

Added new credential service

###Running template Link

**GitHub.com Available Seats**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e7c75e1e3ec660001710d59

**GitHub.com Repositories without Admin Team**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e96f4b7e8dac600012ecbbb

**GitHub.com Repository Branches without Protection**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e7c76bee3ec660001710d5b

**GitHub.com Unpermitted Outside Collaborators**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e7c7ae95315750001ddb826

**GitHub.com Unpermitted Repository Names**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e82e7cb5315750001ddbe17

**GitHub.com Unpermitted Sized Repositories**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e7c8ef9e3ec660001710d6a

**GitHub.com Unpermitted Top-Level Teams**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e82e7855315750001ddbe16

**Okta Inactive Users**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e81954fe3ec6600017111b3

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/reserved_instances/recommendations/README.md [hunk 1]
```diff
@@ -23,26 +17,32 @@ This policy has the following input parameters required when launching the polic
...
+credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed: 
 
-- Cloud Management - Actor
-- Cloud Management - Observer
-- Cloud Management - credential_viewer
+Provider tag value to match this policy: `azure`
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/README.md:36]: Azure EA policies use the provider tag azure_ea

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt [hunk 2]
```diff
@@ -47,11 +46,11 @@ parameter "param_email" do
...
-auth "azure_ea_auth", type: "api_key" do
-  location "header"
-  field "Authorization"
-  type "Bearer"
-  key cred("AZURE_EA_KEY")
+credentials "azure_ea_auth" do
+  schemes "api_key"
+  label "Azure"
+  description "Select the Azure Credential from the list."
+  tags "provider=azure"
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt:53]: should be provider=azure_ea

File: cost/azure/reserved_instances/utilization/README.md [hunk 3]
```diff
@@ -1,46 +1,45 @@
...
+credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed: 
 
-- Cloud Management - Actor
-- Cloud Management - Observer
-- Cloud Management - credential_viewer
+Provider tag value to match this policy: `azure`
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/utilization/README.md:33]: same as above. should be azure_ea

File: cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt [hunk 4]
```diff
@@ -36,11 +35,11 @@ parameter "param_email" do
...
-auth "azure_ea_auth", type: "api_key" do
-  location "header"
-  field "Authorization"
-  type "Bearer"
-  key cred("AZURE_EA_KEY")
+credentials "azure_ea_auth" do
+  schemes "api_key"
+  label "Azure"
+  description "Select the Azure Credential from the list."
+  tags "provider=azure"
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt:42]: same as above, should be azure_ea

File: cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt [hunk 5]
```diff
@@ -36,11 +35,11 @@ parameter "param_email" do
...
 
-auth "azure_ea_auth", type: "api_key" do
-  location "header"
-  field "Authorization"
-  type "Bearer"
-  key cred("AZURE_EA_KEY")
+credentials "azure_ea_auth" do
+  schemes "api_key"
+  label "Azure"
+  description "Select the Azure Credential from the list."
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt:41]: Select the Azure EA Credential

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt [hunk 6]
```diff
@@ -47,11 +46,11 @@ parameter "param_email" do
...
 
-auth "azure_ea_auth", type: "api_key" do
-  location "header"
-  field "Authorization"
-  type "Bearer"
-  key cred("AZURE_EA_KEY")
+credentials "azure_ea_auth" do
+  schemes "api_key"
+  label "Azure"
+  description "Select the Azure Credential from the list."
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt:52]: Select the Azure EA Credential

File: cost/azure/reserved_instances/recommendations/README.md (hunk 1 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/recommendations/README.md]: Updated the provider tag.

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt (hunk 2 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt]: Updated the provider tag.

File: cost/azure/reserved_instances/utilization/README.md (hunk 3 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/utilization/README.md]: Updated the provider tag.

File: cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt (hunk 4 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt]: Updated the provider tag.

File: cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt (hunk 5 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt]: Updated the provider tag.

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt (hunk 6 above)
Comment (KTebrewal) [reply cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt]: Updated the provider tag.

**Pull request merged**